tests/testthat/helper-data.R

```diff
@@ -0,0 +1,55 @@
+# Re-usable data for sample ADSL table
```

Comment:
I've changed the variables names to follow convention of other tests, so that it can be easier integrated in the future.